SurveyApp/Modules/Login/LoginViewController.swift

```diff
@@ -33,15 +43,126 @@ final class LoginViewController: UIViewController {
+        view.addSubview(emailField)
+        view.addSubview(passwordField)
+        view.addSubview(forgotButton)
+        view.addSubview(loginButton)
+
+        backgroundImageView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+            $0.width.equalTo(UIScreen.main.bounds.width)
+        }
+
+        logoImageView.snp.makeConstraints {
+            $0.top.equalToSuperview().offset(150.0)
+            $0.centerX.equalToSuperview()
+            $0.height.equalTo(40.0)
+            $0.width.equalTo(180.0)
+        }
+
+        emailField.snp.makeConstraints {
+            $0.top.equalTo(logoImageView.snp.bottom).offset(120.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+
+        passwordField.snp.makeConstraints {
+            $0.top.equalTo(emailField.snp.bottom).offset(20.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+
+        forgotButton.snp.makeConstraints {
+            $0.top.equalTo(emailField.snp.bottom).offset(20.0)
+            $0.trailing.equalTo(passwordField.snp.trailing)
+            $0.leading.equalToSuperview().inset(280.0)
+            $0.height.equalTo(55.0)
+        }
+
+        loginButton.snp.makeConstraints {
+            $0.top.equalTo(passwordField.snp.bottom).offset(20.0)
+            $0.leading.trailing.equalToSuperview().inset(24.0)
+            $0.height.equalTo(55.0)
+        }
+    }
+    
     private func setUpViews() {
-        view.backgroundColor = .white
+        let backgroundImage = R.image.loginBackground()
+        let logoImage = R.image.logo()
+
+        navigationController?.isNavigationBarHidden = true
+
+        backgroundImageView.image = backgroundImage
+
+        logoImageView.image = logoImage
+        logoImageView.contentMode = .scaleAspectFit
+
+        forgotButton.backgroundColor = .clear
+        forgotButton.setTitle("Forgot?", for: .normal)
+        forgotButton.titleLabel?.font = .regular(ofSize: .body)
+        forgotButton.tintColor = UIColor.white.withAlphaComponent(0.3)
+
+        loginButton.backgroundColor = .white
+        loginButton.setTitle("Log in", for: .normal)
+        loginButton.titleLabel?.font = .bold(ofSize: .body)
+        loginButton.tintColor = .black
+        loginButton.layer.cornerRadius = 10.0
+        
+        setUpTextField()
+        setUpBlurOverlayView()
+    }
+    
+    private func setUpTextField() {
+        emailField.placeholder = "Email"
+        emailField.placeholderColor = UIColor.white.withAlphaComponent(0.3)
+        emailField.layer.cornerRadius = 12.0
+        emailField.textColor = .white
+        emailField.backgroundColor = UIColor.white.withAlphaComponent(0.18)
+        emailField.keyboardType = .emailAddress
+        emailField.autocorrectionType = .no
+        emailField.autocapitalizationType = .none
+        emailField.font = .regular(ofSize: .body)
```

Comment:
Add all of this in TextFieldWithPadding so you dont have to repeat it for both textfields

Reply:
Thank you for pointing out my mistake. I'll make that change! 🙏